README.md

```diff
@@ -86,6 +86,12 @@ Or
 yarn install
 ```
 
+**Note:** If the above installation didn't worked for you try running: 
```

Comment:
How about this?

**Note**: `ReactPlay` runs on React 18. However, some of our dependencies are yet to upgrade to version 18. So please use the following command when you face difficulties installing the dependencies.

```bash
npm install --legacy-peer-deps
```

Reply:
Better than my implementation, I will change it to this then

Reply:
Also, please use it is a quote

> **Note**: ReactPlay runs on React 18. However, some of our dependencies are yet to upgrade to version 18. So please use the following command when you face difficulties installing the dependencies.
  
  ```bash
    npm install --legacy-peer-deps
  ```

@nirban256